### DISCUSSION TITLE
Is it possible to run the .exe silently?

### DISCUSSION THREAD

**Discussion by BishDub** [martinet101/ElevenClock#1011] [Q&A]
Is it possible to run the .exe silently?

**Comment by marticliment** [comment]
What do you mean?

**Reply by BishDub** [reply]
I've downloaded the legacy installer (exe) and wondered if there was some command line switches to install it without the installation wizard. Some examples: .\installer.exe /silent /quiet /qn

**Reply by marticliment** [reply]
Yes, the installer is based on inno, so the switches you need are: `/verysilent /unattended /supressmsgboxes`

**Reply by BishDub** [reply]
Brilliant, thanks for the help.
